plugins/example-code-snippets/src/examples/resolvers/source-parsers/languages/typescript-snippet-source-parser.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  RegexSnippetSourceParser,
+  RegexSnippetTypeOptions,
+} from '../regex-snippet-source-parser';
+import {ExampleSnippetType} from '../../../examples';
+import * as path from 'path';
+
+export class TypescriptSnippetSourceParser extends RegexSnippetSourceParser {
+  constructor(repoSourceDir: string) {
+    const codeSnippetFiles: Array<string> = [
+      // For now we don't have any typescript examples, only javascript ones.  We
+      // can add a list of files containing TS examples in the future if we have some.
```

Comment:
I don't understand this comment. The example file in the SDK repo is in typescript, and the JavaScript snippet parser references a `.ts` file. Hence we do have TypeScript examples and they are being called JavaScript?

Reply:
lol as long as the snippets don't include any type annotations, they are valid JS.  If this proves problematic I can change that file to be .js in the future but that would require monkeying with the build scripts in order to make sure that it gets run during CI so I'm just trying to do the fastest/simplest thing for now.